### PR TITLE
Group calibration variables on config

### PR DESCRIPTION
**Pull request by cua-cua**
Pending on mmlib pull request.

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#include "config.h"
+
+static volatile float micrometers_per_count = MICROMETERS_PER_COUNT;
+static volatile float wheels_separation = WHEELS_SEPARATION;
+static volatile struct control_constants control = {
+    KP_LINEAR,	KD_LINEAR,	KP_ANGULAR,      KD_ANGULAR,
+    KP_ANGULAR_FRONT, KI_ANGULAR_FRONT, KP_ANGULAR_SIDE, KI_ANGULAR_SIDE};
```
Comment (Peque) [inline src/config.c]: I would use this syntax (check before using it, might be wrong, but there is something similar to specify the struct field name):

```
.kp_linear = KP_LINEAR, ...
```

File: src/config.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+static volatile float run_max_linear_speed = RUN_MAX_LINEAR_SPEED;
```
Comment (Peque) [inline src/config.c]: linear_speed_limit?

File: src/config.c (hunk 1 above)
Comment (cua-cua) [reply src/config.c]: 70cf7e59

File: src/config.c (hunk 2 above)
Comment (cua-cua) [reply src/config.c]: 70cf7e59

**Pull request merged**